src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConfig.java

```diff
@@ -19,6 +22,9 @@ public class MarkLogicSourceConfig extends MarkLogicConfig {
     public static final String DSL_QUERY = "ml.source.optic.dsl";
     public static final String SERIALIZED_QUERY = "ml.source.optic.serialized";
     public static final String CONSTRAINT_COLUMN_NAME = "ml.source.optic.constraintColumn.name";
+    public static final String CONSTRAINT_STORAGE_URI = "ml.source.optic.constrainingColumn.uri";
```

Comment:
This is correct per the story, but the story is now a little out-of-date (no need to update it) since we landed on "constraintColumn" above. So for consistency, these should all be "constraintColumn" too.